### PR TITLE
Merge 5 -> 6 

### PR DESCRIPTION
**Pull request by iche033**
# ➡️ Forward port

Port `ign-rendering5` to `main`

Branch comparison: https://github.com/ignitionrobotics/ign-rendering/compare/main...ign-rendering5

**Note to maintainers**: Remember to **Merge** with commit (not squash-merge or rebase)

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -15,7 +15,7 @@ set(IGN_CMAKE_VER ${ignition-cmake2_VERSION_MAJOR})
 #============================================================================
 # Set up the project
 #============================================================================
-ign_configure_project(VERSION_SUFFIX pre1)
+ign_configure_project(VERSION_SUFFIX)
```
Comment (chapulina) [inline CMakeLists.txt:18]: We should keep the `pre1` on `main` so the nightlies are versioned correctly. We only remove this when we're ready for a stable release.

Comment (iche033) [reply CMakeLists.txt:18]: fixed. 07555cc

**Comment by ahcorde** [comment]
@iche033  Do you mind to sign last 3 commits ?

**Comment by iche033** [comment]
> @iche033 Do you mind to sign last 3 commits ?

DCO should be fixed now

**Review by chapulina — APPROVED** [review]
It would be nice to squash the last 3 commits before merging to keep the history clean.

I'm curious why this PR has 17 commits, but the branch comparison has only 7. The PR shows commits that are already on `main`, like e81c974 renamed to 9e8f574  :monocle_face: Git can be very weird :woman_shrugging:

**Comment by iche033** [comment]
> I'm curious why this PR has 17 commits, but the branch comparison has only 7. The PR shows commits that are already on main, like e81c974 renamed to 9e8f574 monocle_face Git can be very weird woman_shrugging

 I think this started when I ran the commands to sign the last 10 commits as instructed by DCO. Now if I squash and force push, I get conflicts, not sure why. Then I have to merge with `main` and resolve conflicts, and that adds 1 more commit.

**Review by ahcorde — APPROVED** [review]

**Comment by chapulina** [comment]
Ok, I think I fixed it. The PR now has 8 commits, the 7 from the diff, plus the merge commit. I fixed the `~pre1` during the merge, so there's no extra commit for that.

Here's what I did:

```
git checkout merge_5_6_20210426
git reset --hard main
git merge ign-rendering5
# fix conflicts and pre1
git commit -sam"5 ➡️  main"
git push --force-with-lease
```

**Comment by iche033** [comment]
cool thanks, looks good to me!

**Comment by iche033** [comment]
something went wrong on the windows build. Retriggering build

**Comment by iche033** [comment]
@osrf-jenkins run tests please

**Pull request merged**